core/src/test/java/io/aiven/klaw/service/TopicControllerServiceTest.java

```diff
@@ -336,16 +566,74 @@ public void approveTopicRequestsSuccess() throws KlawException {
     when(clusterApiService.approveTopicRequests(
             anyString(), anyString(), anyInt(), anyString(), anyString(), any(), anyInt()))
         .thenReturn(new ResponseEntity<>(apiResponse, HttpStatus.OK));
-    when(manageDatabase.getTeamsAndAllowedEnvs(anyInt(), anyInt()))
-        .thenReturn(Collections.singletonList("1"));
+    when(commonUtilsService.getEnvsFromUserId(anyString()))
+        .thenReturn(new HashSet<>(Collections.singletonList("1")));
 
     ApiResponse apiResponse1 = topicControllerService.approveTopicRequests(topicId + "");
+    assertThat(apiResponse1.getResult()).isEqualTo(ApiResultStatus.SUCCESS.value);
+  }
 
+  @Test
+  @Order(24)
+  public void approveTopicClaimRequestsSuccess2() throws KlawException {
+    String topicName = "topic1";
+    int topicId = 1001;
+    TopicRequest topicRequest = getTopicRequest(topicName);
+    topicRequest.setTopictype(TopicRequestTypes.Claim.name());
+    ApiResponse apiResponse = ApiResponse.builder().result(ApiResultStatus.SUCCESS.value).build();
+
+    stubUserInfo();
+    when(handleDbRequests.selectTopicRequestsForTopic(anyInt(), anyInt())).thenReturn(topicRequest);
+    when(handleDbRequests.updateTopicRequest(any(), anyString()))
+        .thenReturn(ApiResultStatus.SUCCESS.value);
+    when(handleDbRequests.getTopicTeam(anyString(), anyInt()))
+        .thenReturn(List.of(getTopic(topicName)));
+    when(clusterApiService.approveTopicRequests(
+            anyString(), anyString(), anyInt(), anyString(), anyString(), any(), anyInt()))
+        .thenReturn(new ResponseEntity<>(apiResponse, HttpStatus.OK));
+    when(commonUtilsService.getEnvsFromUserId(anyString()))
+        .thenReturn(new HashSet<>(Collections.singletonList("1")));
+    when(commonUtilsService.getFilteredTopicsForTenant(any()))
+        .thenReturn(List.of(getTopic(topicName)));
+    when(handleDbRequests.addToSynctopics(any())).thenReturn(ApiResultStatus.SUCCESS.value);
+    when(handleDbRequests.updateTopicRequestStatus(any(), anyString()))
+        .thenReturn(ApiResultStatus.SUCCESS.value);
+
+    ApiResponse apiResponse1 = topicControllerService.approveTopicRequests(topicId + "");
     assertThat(apiResponse1.getResult()).isEqualTo(ApiResultStatus.SUCCESS.value);
   }
 
   @Test
-  @Order(14)
+  @Order(25)
+  public void approveTopicUpdateRequestsSuccess3() throws KlawException {
```

Comment:
We should give it a proper name instead of `1` and `2`.

Reply:
Agree, updated.